#Message where each sub command message for the MQS is published into

uint8[16] cmds
